### PR TITLE
fix: add `frontend/node_modules` to gitignore

### PR DESCRIPTION
**Pull request by Unix4ever**
This folder is now created by `dev-server` command, so it should be
added to `.gitignore`.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: .conform.yaml [hunk 1]
```diff
@@ -28,12 +28,10 @@ policies:
...
     includeSuffixes:
       - .go
     excludeSuffixes:
       - .pb.go
       - .pb.gw.go
-    header: |
-      // This Source Code Form is subject to the terms of the Mozilla Public
-      // License, v. 2.0. If a copy of the MPL was not distributed with this
-      // file, You can obtain one at http://mozilla.org/MPL/2.0/.
+    header: "// This Source Code Form is subject to the terms of the Mozilla Public\u000A// License, v. 2.0. If a copy of the MPL was not distributed with this\u000A// file, You can obtain one at http://mozilla.org/MPL/2.0/.\u000A"
```
Comment (rsmitty) [inline .conform.yaml:37]: Assuming this will still get injected just fine on the source files, even though this format has changed?

**Review by rsmitty — APPROVED** [review]

File: .conform.yaml (hunk 1 above)
Comment (Unix4ever) [reply .conform.yaml:37]: yep

**Pull request merged**